CMSgov/design-system
# [WNMGDS-699] Fix components top level page

## Summary
If you try and go to https://design.cms.gov/components/ you will see the left navigation is different than what is seen on the live site at https://design.cms.gov

http://design-system-demo.s3-website-us-east-1.amazonaws.com/WNMGDS-699/components-top-level-page/components

### Added
List new features or components. Include a screenshot for new visual elements.

### Changed
In the way the design scripts were creating pages, there was no real way to create a top level `https://design.cms.gov/components/` page. I have added an index.md to the components directory to act as the top level components page (with the content previously seen at https://design.cms.gov/components/), and adjusted the scripts so that when instead of creating `https://design.cms.gov/components/index` it creates the top level `https://design.cms.gov/components/`

## How to test
- First run everything on master. Check http://localhost:3000/components and you'll get nothing
- Run on this branch and build, you should get a top level page with the left sidebar correct
